src/Essentials/samples/Samples/Platforms/Android/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:installLocation="auto">
-	<uses-sdk android:minSdkVersion="21" android:targetSdkVersion="31" />
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" xmlns:tools="http://schemas.android.com/tools" android:installLocation="auto">
+	<uses-sdk android:minSdkVersion="21" android:targetSdkVersion="31" tools:overrideLibrary="androidx.security"/>
```

Comment:
@jonathanpeppers this is what @mattleibow was asking you for me about! wondering what a better way of doing this might be

Reply:
@dellis1972, can you help look at this next week? I tried to mark you reviewer, but it won't let me.

Reply:
I wonder if we can leverage the new generated aar to include a AndroidManifest.xml? @dellis1972 Is that possible, and can a library override/implement this type of attribute?

Reply:
Is this `overrideLibrary` attribute being included because of this? (from [Manifest Merger](https://android-doc.github.io/tools/building/manifest-merge.html#merge-rules) ) 

```
Importing a library with a minSdkVersion value higher than the app's src/main/ manifest manifest generates an error unless the overrideLibrary conflict marker is used.
```
which suggests that `androidx.security` is specifying a `minSdkVersion` which is higher than `21`.  In which case should we be using that library? What will happen on Api 21 devices when they try to use the `androidx.security` features? I suspect we might get a runtime error.

If we can use that library then in theory including an AndroidManifest.xml with the required additional attributes should be handled by the manifest merger. 
In this case, the maui android "platform" library would need to include an AndroidManifest.xml which just contains

```xml

<manifest xmlns:android="http://schemas.android.com/apk/res/android" xmlns:tools="http://schemas.android.com/tools" >
	<uses-sdk  tools:overrideLibrary="androidx.security"/>
</manifest>
```

that should end up getting merged in automatically (assuming that is the behaviour we want).

Reply:
Perhaps the easiest way out of this warning would be to get bindings for the latest version of this package. Unfortunately there is a newer version marked alpha which works with 21 explicitly (and would simplify our code), but.  It's marked alpha. As with many of these libs google seems to have unstable versions yet refer to using them as if they are stable in their conceptual docs.